src/main/java/uk/gov/hmcts/probate/services/persistence/controllers/SequenceNumberController.java

```diff
@@ -8,7 +9,7 @@
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RestController;
-import uk.gov.hmcts.probate.services.persistence.services.SequenceNumberService;
+import uk.gov.hmcts.probate.services.persistence.services.RegistrySequenceService;
 
 @RestController
 public class SequenceNumberController {
```

Comment:
Hi Tom, thinking about it, I think the corrosponding SequenceNumberController (and associated tests) would need to be renamed as well to match the naming convention used by the RegistrySequenceService.